ui/src/store/account.js

```diff
@@ -67,10 +71,26 @@ export default {
 		},
 
 		async login({ commit }, { email, password }) {
-			const { token } = await login({
-				email,
-				password
-			});
+			let token;
+
+			try {
+				commit("setErrorMessage", { message: "" });
+
+				const response = await login({
+					email,
+					password
+				});
+
+				token = response.token;
+			} catch (e) {
+				if (e instanceof SatelliteError) {
+					commit("setErrorMessage", { message: e.message });
+				} else {
+					console.log(e);
```

Comment:
Can we remove the else condition, then have `throw e` instead of return?

Reply:
@montyanderson Are you saying something like this?
```
} catch (e) {
  if (e instanceof SatelliteError) {
      commit("setErrorMessage", { message: e.message });
  }

  console.log(e);
  throw e;
}
```

If we're throwing this error here, are you thinking that we'll catch it in the `login` component?

Reply:
No log, just throw